src/Api/State/BeneficiaryStateProcessor.php

```diff
@@ -43,7 +43,7 @@ public function process(mixed $data, Operation $operation, array $uriVariables =
 
         if ($data instanceof LinkBeneficiaryDto && $operation instanceof Patch && $client) {
             $beneficiary = $this->beneficiaryRepository->find($uriVariables['id']);
-            if (!$beneficiary->getExternalLinksForClient($client)->isEmpty()) {
+            if (!$beneficiary->getExternalLinksForClient($client)->isEmpty() && Client::CLIENT_RECONNECT_PRO !== $client->getNom()) {
```

Comment:
If we check this in 2 conditions (Beneficiaire entity too), maybe we could have a method in Client like `isMultipleLinksAllowed()`